src/components/custom/platforms-dropdown.js

```diff
@@ -77,47 +67,36 @@ const PlatformsDropdown = ({
 }) => {
     const dropdownContainerRef = useRef(null)
     const is_trade = active_dropdown === 'trade'
-    const current_offset = current_ref.offsetWidth / 2 - 15
-    const setTradeArrowOffset = (dropdownOffset) =>
-        current_ref.offsetLeft - dropdownOffset + current_offset
-    const [left_offset, setLeftOffset] = useState(current_ref.offsetLeft)
-    const [left_arrow_offset, setLeftArrowOffset] = useState()
+
+    const [left_offset, setLeftOffset] = useState(() => {
+        if (is_trade) {
+            return current_ref?.getBoundingClientRect()?.x / 2
```

Comment:
Hi @michio-rework, just wanna ask what this part of the code is doing to understand it better ? 🙏

Reply:
Hi @mitra-deriv , please check [this](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) out.

I'm using `X` coordinates of the clicked `li` element as `left` property of dropdown ( the absolute div which is being opened ). 

But for the first `li` ( Trade ) we have show it differently since based on ROW and etc we have different items in dropdown and eventually the size of it will be bigger. in smaller screen if we use the `X` coordinate of the `li` when region is ROW it goes our of screen sometimes. so we have to use half of `X` coordinate of clicked `li` to make sure the content is matched with the desing. 

If you have more questions please dm me and we can discuss this furthur.